src/enums/definitions.json

```diff
@@ -1535,68 +1535,94 @@
     ]
   ],
   "TRANSACTION_RESULTS": {
-    "telNO_DST_PARTIAL": -393,
-    "temBAD_SRC_ACCOUNT": -281,
-    "tefPAST_SEQ": -189,
-    "terNO_ACCOUNT": -96,
-    "temREDUNDANT": -275,
-    "tefCREATED": -194,
-    "temDST_IS_SRC": -279,
-    "terRETRY": -99,
-    "temINVALID_FLAG": -276,
-    "temBAD_SEND_XRP_LIMIT": -288,
-    "terNO_LINE": -94,
-    "tefBAD_AUTH": -196,
-    "temBAD_EXPIRATION": -295,
-    "temBAD_SEND_XRP_NO_DIRECT": -286,
-    "temBAD_SEND_XRP_PATHS": -284,
-    "tefBAD_LEDGER": -195,
-    "tefNO_AUTH_REQUIRED": -190,
-    "terOWNERS": -93,
-    "terLAST": -91,
-    "terNO_RIPPLE": -90,
-    "temBAD_FEE": -294,
-    "terPRE_SEQ": -92,
-    "tefMASTER_DISABLED": -187,
-    "temBAD_CURRENCY": -296,
-    "tefDST_TAG_NEEDED": -193,
-    "temBAD_SIGNATURE": -282,
-    "tefFAILURE": -199,
-    "telBAD_PATH_COUNT": -397,
-    "temBAD_TRANSFER_RATE": -280,
-    "tefWRONG_PRIOR": -188,
+    "telLOCAL_ERROR": -399,
     "telBAD_DOMAIN": -398,
+    "telBAD_PATH_COUNT": -397,
+    "telBAD_PUBLIC_KEY": -396,
+    "telFAILED_PROCESSING": -395,
+    "telINSUF_FEE_P": -394,
+    "telNO_DST_PARTIAL": -393,
+    "telCAN_NOT_QUEUE": -392,
+    "telCAN_NOT_QUEUE_BALANCE": -391,
+    "telCAN_NOT_QUEUE_BLOCKS": -390,
+    "telCAN_NOT_QUEUE_BLOCKED": -389,
+    "telCAN_NOT_QUEUE_FEE": -388,
+    "telCAN_NOT_QUEUE_FULL": -387,
+
+    "temMALFORMED": -299,
     "temBAD_AMOUNT": -298,
     "temBAD_AUTH_MASTER": -297,
```

Comment:
`temBAD_AUTH_MASTER` is not currently one of the `tem` codes.  In the rippled source `temBAD_CURRENCY` is the code that follows `temBAD_AMOUNT`.  In the rippled source code the numeric value of `temBAD_CURRENCY` is -297.